other-guides/hosting-on-linux-server

```diff
@@ -0,0 +1,53 @@
+# Hosting on a Linux Server
+
+The best way to host your bot is on a Linux Server (or simply said: just a virtual server. Its also for bare metal servers).
+
+## Requirements
+
+- A Linux Server with a stable version, node, npm, git, pm2. 
+- Github repository from your bot (privat).
+- SSH Client
+
+## Getting Started
+
+### Login
+
+The best way to do that is with root access. If its root access the username would be `root`.
```

Comment:
While the default user is `root`, it's very unwise to use it, as a vulnerability in your bot or one of its services can put the entire system at risk.

Please recommend creating a new user with a password different to `root`.